### DISCUSSION TITLE
yarn link equivalent ?

### DISCUSSION THREAD

**Discussion by mdjfs** [expo/expo#26285] [Proposals and Ideas]
Hello! I was trying to test a local library in an Expo App. But seems yarn link didn't work. When i try start the app gives me the error "Unable to resolve..." Some way to do this stuff? Is very complicated develop something like `expo link <some-package>`?

**Comment by NaiySliogan** [comment]
I have the exact same issue. Did you find a solution ?

**Comment by mdjfs** [comment]
Hello! Not really, just a work around. Build the library and copy the files to `node_modules/<some-package>`. Some other ways may be complicated because metro don't know about your filesystem

**Reply by NaiySliogan** [reply]
Hello ! 
Thank you for your answer.

If you are interested I have finally found a solution that worked for me with expo 49. 

This metro config worked for me


```
metro.config.js


/* eslint-disable */
// Learn more https://docs.expo.dev/guides/monorepos
const { getDefaultConfig } = require('expo/metro-config')
const path = require('path')

// Find the project and workspace directories
const projectRoot = __dirname
// This can be replaced with `find-yarn-workspace-root`
const workspaceRoot = path.resolve(projectRoot, '../..')

const config = getDefaultConfig(projectRoot)

const monorepoPackages = {
  '@some-library':   path.resolve(workspaceRoot, 'node_modules/@some-library/src'),
}

config.watchFolders = [workspaceRoot, ...Object.values(monorepoPackages)]

config.resolver.extraNodeModules = monorepoPackages

config.resolver.unstable_enableSymlinks = true

config.resolver.nodeModulesPaths = [
  path.resolve(projectRoot, 'node_modules'),
  path.resolve(workspaceRoot, 'node_modules'),
]

module.exports = config
```

If you want to try it don't forget to clear the cache with `expo start --clear`
